packages/graphql/package.json

```diff
@@ -28,6 +28,7 @@
   },
   "peerDependencies": {
     "graphql-tag": "^2.5.0",
+    "react": ">=15.6.2 <17.0.0",
```

Comment:
Why not `>=15.0.0`? This now seems to imply that something changed in `15.6.2` and we can't support previous versions.